great_expectations/data_context/types/base.py

```diff
@@ -505,6 +543,21 @@ def validate_schema(self, data, **kwargs):
                     You must only select one between `conn_str` and `account_url`. Please update your configuration to continue.
                     """
                 )
+        if (
+            "gcs_options" in data or "bucket_or_name" in data or "max_results" in data
+        ) and not (
+            data["class_name"]
+            in [
+                "InferredAssetGCSDataConnector",
+                "ConfiguredAssetGCSDataConnector",
+            ]
+        ):
+            raise ge_exceptions.InvalidConfigError(
+                f"""Your current configuration uses one or more keys in a data connector, that are required only by an
```

Comment:
@cdkini either delete the comma or use "which" in place of "that".  Thanks!

Reply:
Changed all instances of this in the file!